routes/web.php

```diff
@@ -99,5 +99,5 @@
     Route::post('/integrations/{id}/coupon', [IntegrationController::class, 'activateWithCoupon']);
     Route::post('/integrations/{id}/organization', [IntegrationController::class, 'activateWithOrganization']);
 
-    Route::get('/integrations/{id}/widget', [IntegrationController::class, 'showWidget']);
+    Route::get('/integrations/{integration}/widget', [IntegrationController::class, 'showWidget']);
```

Comment:
What is this vodoo magic? 😛 Is Laravel able to automatically convert id to `integrationModel`?

Reply:
It is called Route Model Binding: https://laravel.com/docs/10.x/routing#route-model-binding

Reply:
Nice 💪🏻